### ISSUE TITLE
Bundler could not find compatible versions for gem "activemodel"

### ISSUE THREAD

**Issue by danielbonnell** [magnusvk/counter_culture#163]
Hi, I am currently running `v1.1.1` with Rails `4.2.7.1`. When attempting to update to `v1.3.1` I am encountering the following error:

```
Fetching gem metadata from https://rubygems.org/........
Fetching version metadata from https://rubygems.org/..
Fetching dependency metadata from https://rubygems.org/.
Resolving dependencies.............
Bundler could not find compatible versions for gem "activemodel":
  In Gemfile:
    active_model_serializers (= 0.10.4) was resolved to 0.10.4, which depends on
      activemodel (< 6, >= 4.1)

    counter_culture (~> 1.3.1) was resolved to 1.3.1, which depends on
      activerecord (>= 3.0.0) was resolved to 5.0.1, which depends on
        activemodel (= 5.0.1)

    rails (= 4.2.7.1) was resolved to 4.2.7.1, which depends on
      activemodel (= 4.2.7.1)
```

I may have misread the changelog, but I was under the impression that `v1.3.x` is backwards compatible with older versions of Rails?

Thanks!

**Comment by magnusvk** [comment]
@danielbonnell yes, 1.3.1 is compatible with older versions of Rails. In fact, that's stated right there in your snippet above:
```
counter_culture (~> 1.3.1) was resolved to 1.3.1, which depends on
      activerecord (>= 3.0.0) was resolved to 5.0.1, which depends on
        activemodel (= 5.0.1)
```

Which says counter_culture is looking for activerecord >= 3.0.0. May this just be a bundler problem? Are you doing a full-on bundle update here?

But yes, this should definitely work for you.

**Comment by danielbonnell** [comment]
This is pretty odd. I'm running Bundler `1.14.5` (latest) and if I try `bundle update` or `bundle update counter_culture`, they both failed. I decided to try adding `gem 'activerecord', '4.2.7.1'` to my Gemfile even though it's already listed in my Gemfile.lock. Re-ran `bundle update counter_culture` and it worked. Then I went back, removed `gem 'activerecord', '4.2.7.1'` from my Gemfile and ran `bundle` again to see if it broke, but it didn't. Obviously something wrong with Bundler. Thanks!
